File: 백준/Bronze/1152. 단어의 개수/단어의 개수.py
N = str(input())
j = 0
count = 0
for i in range(0,len(N)):

    if j == 0 and N[i] != " ":
        count += 1
        j = 1
    if N[i] == " ":
        j = 0
print(count)    

File: 백준/Bronze/1152. 단어의 개수/단어의 개수.cs
using System;
using System.Diagnostics.PerformanceData;
using System.Runtime.InteropServices;

namespace baekjoon
{
    class Program
    {
        static void Main(string[] args)
        {
            int count = 0;
            string[] s = Console.ReadLine().Split();
            for(int i=0; i < s.Length;i++)
            {
                if (s[i] != "")
                {
                    count += 1;
                }
            }
            Console.WriteLine(count);
        }
    }
}